Nodejs/Product/Nodejs/Snippets/1033/Nodejs/tryc.snippet

```diff
@@ -0,0 +1,31 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>tryc</Title>
+      <Description>Code Snippet for a Try...Catch Statement</Description>
+      <Shortcut>tryc</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>

```

Comment:
Make this a SurroundsWith as well - I anticipate that'll be the most likely scenario anyways.